modular_skyrat/master_files/code/modules/pain_shock/status_indicator.dm

```diff
@@ -0,0 +1,198 @@
+#define PLANE_STATUS -12 //Status Indicators that show over mobs' heads when certain things like stuns affect them.
+
+#define VIS_STATUS			24
+
+#define VIS_COUNT			24 //Must be highest number from above.
+#define STATUS_LAYER -2.1
+#define STATUS_INDICATOR_Y_OFFSET 2 // Offset from the edge of the icon sprite, so 32 pixels plus whatever number is here.
+#define STATUS_INDICATOR_ICON_X_SIZE 0 // Don't need to care about the Y size due to the origin being on the bottom side.
+#define STATUS_INDICATOR_ICON_MARGIN 0 // The space between two status indicators. We don't do this with the current icons.
+
+/datum/preference/toggle/enable_status_indicators
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	savefile_key = "enable_status_indicators"
+	savefile_identifier = PREFERENCE_PLAYER
+
+/datum/preference/toggle/enable_status_indicators/create_default_value()
+	return TRUE
+
+/datum/preference/toggle/enable_status_indicators/apply_to_client(client/client, value)
+	. = ..()
+	var/atom/movable/screen/plane_master/runechat/status/status = locate() in client?.screen
+	if(!status)
+		return
+	if(value)
+		status.alpha = 255
+	else
+		status.alpha = 0
```

Comment:
I'm not entirely sure why, but I've got a sneaky suspicion this isn't exactly working, considering I've had my pref off since it's been added, and I still see them now.

Reply:
Think I should delete this layer from mobs instead, and recreate?